CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+### citus v12.1.3 (April 18, 2024) ###
+
+* Allows overwriting host name for all inter-node connections by supporting "host"
+  parameter in citus.node_conninfo (#7541)
+
+* Correctly mark some variables as volatile (#7570)
+
+* Fix performance issue in GetForeignKeyOids on systems with many constraints (#7578)
```

Comment:
Better to have a more user friendly description --users are not expected to know underlying C function names:

```suggestion
* Fixes performance issue in when tracking foreign key constraints on systems with many constraints (#7578)
```

Reply:
Added as you suggested